src/pages/en/migrate.md

```diff
@@ -6,6 +6,28 @@ description: How to migrate your project to latest version of Astro.
 
 Until Astro reaches v1.0, we expect to make some breaking changes across minor versions (ex: `v0.1 -> v0.2`). This guide exists to help you migrate to the latest versions of Astro and keep your codebase up-to-date.
 
+## Migrate to v0.25
+
+### A new integration system
+
+The `renderers` config has been replaced by a new, official integration system! Read our ["Using Integrations"](/en/guides/integrations-guide) guide for more details.
+
+If you already had custom `renderers` defined in your configuration, you will see a notice on startup telling you how to update your configuration file to move from the deprecated `renderers` config to the new `integrations` config, complete with an example code snippet for you to copy.
+
+Note that the integrations have differently names from the older renderer npm packages. Going forward, `@astrojs/renderer-react` becomes `@astrojs/react`, `@astrojs/renderer-vue` becomes `@astrojs/vue`, and so on.
```

Comment:
fwiw, when you run `astro` it will tell you every update you need to make based on your existing `renderers` config. Let me see if I can add that table and then highlight that output.

Reply:
updated the docs to be more clear. Thanks for the feedback so far!